backend/index.js

```diff
@@ -19,4 +20,36 @@ app.get("/", (req, res) => {
   res.send("Hello world!");
 });
 
+// #1 - get all events
+app.get("/api/events", async (req, res) => {
```

Comment:
I think its usually better to omit "api" from api routs
```suggestion
app.get("/events", async (req, res) => {
```